vagrant/Vagrantfile

```diff
@@ -0,0 +1,17 @@
+Vagrant.configure(2) do |config|
+    config.vm.box = "debian/jessie64"
+
+    config.vm.synced_folder ".", "/vagrant", type: "rsync", disabled: true
+
+    config.vm.provider :libvirt do |libvirt|
```

Comment:
I would list the requirements (minimum/recommended) in the README file. 4Gb of RAM is not little.